daemon/daemon.go

```diff
@@ -164,7 +164,7 @@ func managementServerDown() {
 // operationalServerUp starts the operational server if it is
 // not running
 func operationalServerUp() {
-	if server.Running() != server.OPERATIONAL {
+	if server.Current != server.Operational && server.State == server.Stopped {
```

Comment:
They were originally commented out and I didn't realize. Now they are included also